python/mxnet/ndarray/ndarray.py

```diff
@@ -2885,6 +2885,11 @@ def attach_grad(self, grad_req='write', stype=None):
             ctypes.pointer(mx_uint(grad_req)),
             ctypes.pointer(grad.handle)))
 
+    def drop_grad(self):
```

Comment:
Could you also add drop_grad to MXNet NumPy array package.

Reply:
Yes it's added now.